roofit/roofitcore/src/RooRealSumPdf.cxx

```diff
@@ -283,33 +283,26 @@ void RooRealSumPdf::computeBatch(cudaStream_t* stream, double* output, size_t nE
       }
   }
 
-  RooAbsPdf::computeBatch(stream, output, nEvents, dataMapCopy);
-}
-
-
-////////////////////////////////////////////////////////////////////////////////
-/// Calculate the value for all values of the observable in `evalData`.
-RooSpan<double> RooRealSumPdf::evaluateSpan(RooBatchCompute::RunContext& evalData, const RooArgSet* /*normSet*/) const {
   // Do running sum of coef/func pairs, calculate lastCoef.
-  RooSpan<double> values;
+  for (unsigned int j = 0; j < nEvents; ++j) {
```

Comment:
It would be better to use size_t instead of unsigned int here, and same also below, for the case nEvents>2^32.

Reply:
Ok! I'll change this